04_environment/index.tex

```diff
@@ -120,34 +108,31 @@ \subsubsection{Pareto Optimal Strategy}
 
 \subsubsection{Incorporated Changes: Population Density}
 
-Introducing a model that controls the population density of deer allows us to increase the complexity of the foraging function. In short, the deer hunting capacity decreases with decreasing population and increases with an increasing population. This will then influence the maximum deer per hunt parameter(n), which in turn results in a more complex return calculation function (\texttt{\textbf{DtotalReturn}}), which would make it harder for agents to figure out forage tier boundaries to optimise their strategies.
+Introducing a model that controls the population density of deer allows us to increase the complexity of the foraging function. In short, the deer hunting capacity decreases with decreasing population and increases with an increasing population. This will then influence the maximum deer per hunt parameter ($n$), which in turn results in a more complex return calculation function (\texttt{\textbf{DtotalReturn}}), which would make it harder for agents to figure out forage tier boundaries to optimise their strategies.
 
-The population change can be dynamic depending on various environmental factors, fetched from the rest of the system. For example, a disaster could supposedly cause the population density of deer to halve, making it harder to forage and therefore, causing lower returns. For future implementations, the team uses a single species population model and more specifically logistic modelling for the deer and fish population. The model can be described in Equation~\eqref{eq:Population Density}.
+The population change can be dynamic depending on various environmental factors, fetched from the rest of the system. For example, a disaster could supposedly cause the population density of deer to halve, making it harder to forage and therefore, causing lower returns. In this current implementation, a single species population model and more specifically logistic modelling for the deer and fish population has been implemented. 
+The model is described in Equation~\eqref{eq:Population Density}.
 
 \begin{equation}
 \frac{\mathrm{d} P}{\mathrm{~d} t}=k(N-P)
 \label{eq:Population Density}
 \end{equation}
-
 \begin{itemize}
-        \item “$P$” is the total Population. \item “$N$” is the maximum deer/fish population (“carrying capacity” formally).
-        \item “$k$” is the growth coefficient.
-        \item “$t$” is time.
+        \item $P$ is the total population. 
+        \item $N$ is the maximum deer and fish population (carrying capacity \footnote{The maximum population size of a biological species that can be sustained in that specific environment.} formally).
```

Comment:
(carrying capacity\footnote{This refers to the maximum population ....} 

`\footnote{}` directly comes after word -- no space!

Reply:
remove 'formally'

Reply:
in that specific environment ---> in **a** specific environment